### PR TITLE
fix: emit HTML instead of MDX

### PR DESCRIPTION
**Pull request by dstaley**
🎟️ [Asana Task]()

---

## PR Checklist 🚀

- [ ] Conduct thorough self-review.
- [ ] Add or update tests as appropriate.
- [ ] Write a useful description (above) to give reviewers appropriate context.
- [ ] Identify (in the description above) and document (add Asana tasks on [this board](https://app.asana.com/0/1100423001970639/list)) any technical debt that you're aware of, but are not addressing as part of this PR.

### CODE REVIEW

File: packages/remark-plugins/plugins/paragraph-custom-alerts/index.js [hunk 1]
```diff
@@ -35,21 +35,24 @@ export default function paragraphCustomAlertsPlugin() {
...
+                            `g-type-body`,
+                          ],
+                        },
+                      },
                     },
                     [node]
                   )
                 : node
             })
+            console.dir(parent.children, { depth: Infinity })
```
Comment (BRKalow) [inline packages/remark-plugins/plugins/paragraph-custom-alerts/index.js:55]:
```suggestion
```

**Review by BRKalow — APPROVED** [review]
Ship it!

File: .changeset/polite-ghosts-call.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@hashicorp/platform-markdown-utils': minor
```
Comment (BRKalow) [inline .changeset/polite-ghosts-call.md:2]: Should this be a `major` as we're now planning to expose native ESM?

Comment (dstaley) [reply .changeset/polite-ghosts-call.md:2]: We're still on the 0.x.x series so this is technically a major.

**Pull request merged**